src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/normalize/normalize_kernel_selector.cpp

```diff
@@ -3,13 +3,18 @@
 //
 
 #include "normalize_kernel_selector.h"
-#include "normalize_kernel_within_spatial_ref.h"
+
+#include "normalize_kernel_across_spatial_blocked_ref.h"
 #include "normalize_kernel_across_spatial_ref.h"
+#include "normalize_kernel_within_spatial_blocked_ref.h"
+#include "normalize_kernel_within_spatial_ref.h"
 
 namespace kernel_selector {
 normalize_kernel_selector::normalize_kernel_selector() {
     Attach<NormalizeKernelWithinSpatialRef>();
     Attach<NormalizeKernelAcrossSpatialRef>();
+    Attach<NormalizeKernelAcrossSpatialBlockedRef>();
```

Comment:
Does normalize have any specific reason why it has multiple ref kernel? I know that normalize and MVN have multiple ref kernel while others have single ref kernel usually.

Reply:
Yes, in existing reference kernel is a very simple but linear logic, which means that the next index is received from the previous by simple one-operation addition

```
 output_idx += OUTPUT_FEATURE_PITCH;
        input_idx += INPUT0_FEATURE_PITCH;
```

To support blocked layouts the same approach cannot be used and indices are received by
```
    output_idx =  OUTPUT_GET_INDEX(b, f, y, x);
        input_idx =  INPUT0_GET_INDEX(b, f, y, x);
```

So I suppose we would have potentially not critical but performance regression for plane inputs.

I see 3 options for what we can do here:
1. just replace old reference kernel with this new one and confirm that this regression not critical
2. leave at it's presented now
3. rename new kernel as reference one as it supports all the layouts, and old ref kernel will rename to sth like normalize_kernel_plane_layouts.cl to leave this "simple one-operation addition" functionality

Reply:
For me, current implementation is ok. Let's have others opinion.